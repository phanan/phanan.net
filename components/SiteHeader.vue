<template>
  <header>
    <h1>Phan An</h1>
  </header>
</template>

<style lang="scss" scoped>
h1 {
	font: normal 100 128px/128px Lato, sans-serif;
	text-indent: -10px;
  margin-bottom: 48px;

  @media only screen and (max-width: 640px) {
    font-size: 64px;
    line-height: 64px;
    margin-bottom: 24px;
    text-indent: -4px;
  }
}
</style>
