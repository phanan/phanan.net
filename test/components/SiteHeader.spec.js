import { renderToString } from '@vue/server-test-utils'
import SiteHeader from '~/components/SiteHeader'

describe('SiteHeader.vue', () => {
  it('matches snapshot', () => {
    expect(renderToString(SiteHeader)).toMatchSnapshot()
  })
})
