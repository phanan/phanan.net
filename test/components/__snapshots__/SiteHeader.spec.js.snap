// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader.vue matches snapshot 1`] = `
<header>
  <h1>Phan An</h1>
</header>
`;
